frontend/packages/noobaa-storage-plugin/src/components/data-resiliency-card/data-resiliency-card.scss

```diff
@@ -1,10 +1,8 @@
-@import '~@patternfly/patternfly/sass-utilities/colors';
-
 .nb-data-resiliency__dashboard-body {
   display: flex;
   flex-direction: column;
-  height: 15.9em;
   justify-content: center;
+  margin-top: 1.5em;
```

Comment:
the changes in `nb-data-resiliency__dashboard-body` should be synced with `ceph-data-resiliency__dashboard-body`

Reply:
![cephdr](https://user-images.githubusercontent.com/25664409/62463370-78770c00-b7a7-11e9-985d-ab387f4f3440.png)

Reply:
updated!